flair/datasets/__init__.py

```diff
@@ -148,6 +148,7 @@
 from .treebanks import UD_COPTIC
 from .treebanks import UD_IRISH
 from .treebanks import UD_LATVIAN
+from .treebanks import UD_Lithuanian
```

Comment:
Can you capitalize the object name like the other datasets, i.e. "UD_LITHUANIAN" instead of "UD_Lithuanian"

Reply:
Sure